src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_4_layout.html.twig

```diff
@@ -257,7 +257,8 @@
                 {{- label|trans(label_translation_parameters, translation_domain)|raw -}}
             {%- endif -%}
         {%- endif -%}
-        {% block form_label_errors %}{{- form_errors(form) -}}{% endblock form_label_errors %}</{{ element|default('label') }}>
+        </{{ element|default('label') }}>
+        {% block form_label_errors %}{{- form_errors(form) -}}{% endblock form_label_errors %}
```

Comment:
Should be reverted, or is this needed for this PR?

Reply:
Yes, it's necessary. `form_errors` should be placed after `label`, not within it.

Reply:
Ah indeed, thanks for clarification

Reply:
IIRC the errors have been put inside labels for a reason. Maybe @stof remembers the exact reasoning behind this decision.

Reply:
See https://github.com/symfony/symfony/pull/24435

Reply:
IMHO, It's fine to use the proposed code in #24435. However, the current implementation doesn't follow the format.

```html
<label for="name">Name: <span class="hidden">Error message</span></label>
<input id="name" type="text">
<span aria-hidden="true">Error message</span>
```